backend/app/scripts/create_post.py

```
@@ -0,0 +1,377 @@
+        post_ref.set(post.model_dump())
+        
+        # Step 2: Update global post metrics
+        global_post_metrics_ref = database.collection("global-post-metrics").document("LwOyb4Ffxn9Kj1122mqg")
+        metrics_doc = global_post_metrics_ref.get()
+        
+        if not metrics_doc.exists:
+            raise Exception("Global post metrics document does not exist")
+        
+        # Retrieve current metrics
+        current_metrics = metrics_doc.to_dict()
+        if (current_metrics == None) :
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)